src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/form.html.twig

```diff
@@ -405,7 +395,35 @@
     }
 
     var tabTarget = new TabView(),
-        toggler = new Toggler(new JsonStorage(sessionStorage));
+        toggler = new Toggler(new JsonStorage(sessionStorage)),
+        clickHandler = function (element, callback) {
+            Sfjs.addEventListener(element, 'click', function (e) {
+                var target;
+
+                if (!e) {
+                    e = window.event;
+                }
+
+                if (e.target) {
+                    target = e.target;
+                }
+                else if (e.srcElement) {
+                    target = e.srcElement;
+                }
+
+                callback.call(target);
```

Comment:
Any more corrections? I'm asking because on each change 1 made you require a correction and from my side it takes a lot of time to open/close project in IDE again and again to make one more small correction.